lib/services/sshd.pm

```diff
@@ -1,18 +1,27 @@
 # SUSE's openQA tests
 #
-# Copyright 2021 SUSE LLC
+# Copyright 2021-2022 SUSE LLC
 # SPDX-License-Identifier: FSFAP
 # Summary: Package for ssh service tests
 #
 # Maintainer: QE YaST <qa-sle-yast@suse.de>
```

Comment:
maintainer need change to yourself.

Reply:
It seems you should be the second maintainer.

Reply:
done

Reply:
done